appservice/src/IAppSettingsClient.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SlotConfigNamesResource, StringDictionary } from 'azure-arm-website/lib/models';
+
+export interface IAppSettingsClient {
```

Comment:
I assume you'll need to reference this interface outside the package, meaning you should export it here: https://github.com/microsoft/vscode-azuretools/blob/master/appservice/src/index.ts